src/components/layout/PanelChange/index.tsx

```diff
@@ -192,9 +192,22 @@ export const PanelChange: FC<IProps> = ({
                   </div>
                   <span>
                     {((personalFlow || 0) > 0 && (balanceA || 0) > 0) && (
-                    <div className={styles.stream_values}>
-                      {`Runs out on ${streamEnd}`}
-                    </div>
+                      <div>
+                        <div className={styles.stream_values}>
+                          {`Runs out on ${streamEnd}`}
+                        </div>
+                        <div className={styles.stream_values}>
+                          Streamed 
+                          {' '}
+                          <b>
+                            $
+                            {/* @ts-ignore */}
```

Comment:
@bigint Thats not how it works , you need to calculate the amount sent by checking the stream, its flowrates and counting up all of the streamperiods per flowrate change. Also would actually be cool to see the streamed amount getting animated, which could be done in the js side by just getting the last flowrate and then updating the text shown
Here is an example of how your logic fails: Started a stream for 100$ per month, downgraded 20 USDC , it shows I have streamed 20$ already , in actuallity its a cent or so ,see the screenshots:

![Screenshot 2022-02-17 at 13 13 59](https://user-images.githubusercontent.com/77115130/154470139-d5f023db-4c9b-407d-ac18-85bf26f374d4.png)
![Screenshot 2022-02-17 at 13 14 21](https://user-images.githubusercontent.com/77115130/154470185-3bbd062e-ca39-49e7-876d-8a4193cadb5a.png)

Also why are you ignoring typescript here?

Reply:
Haven't had the time to even check the current codebase , but did the changes also break the UI for Launchpad?
![Screenshot 2022-02-17 at 13 17 57](https://user-images.githubusercontent.com/77115130/154470776-fb69d152-7469-41f0-9d0f-f8712ddb52c2.png)